README.md

```diff
@@ -32,6 +32,8 @@ a good network (which you can feed into this program, suddenly making it strong)
 
 # I want to help
 
+To contribute to leela-zero, you need to run autogtp binary.
```

Comment:
I'm adding some suggestions for wording chances that seem naturally to me. feel free to ignore any or all of them.
run \*the\* autogtp

Reply:
ok